src/bmi_cfe.c

```diff
@@ -213,38 +229,53 @@ static const char *output_var_locations[OUTPUT_VAR_NAME_COUNT] = {
         "node",
         "node",
         "node",
-        "node"
+        "node",
+	"domain",
+	"domain",
+	" "
 };
 
 // Don't forget to update Get_value/Get_value_at_indices (and setter) implementation if these are adjusted
 static const char *input_var_names[INPUT_VAR_NAME_COUNT] = {
         "atmosphere_water__liquid_equivalent_precipitation_rate",
-        "water_potential_evaporation_flux"
+        "water_potential_evaporation_flux",
+	"soil__ice_fraction_schaake",
+	"soil__ice_fraction_xinan"
 };
 
 static const char *input_var_types[INPUT_VAR_NAME_COUNT] = {
         "double",
-        "double"
+        "double",
+	"double",
+	"double"
 };
 
 static const char *input_var_units[INPUT_VAR_NAME_COUNT] = {
         "mm h-1", //"atmosphere_water__liquid_equivalent_precipitation_rate"
-        "m s-1"   //"water_potential_evaporation_flux"
+        "m s-1",   //"water_potential_evaporation_flux"
+	"m",    // ice fraction in meters
+	"",     // ice fraction [-]
```

Comment:
`"none"` ?

Reply:
does BMI use "none" for dimensionless quantities?

Reply:
okay, found it (Variables without units should use "none".)